msteams-platform/get-started/first-message-extension.md

```diff
@@ -162,6 +162,23 @@ Deployment involves provisioning resources on an active Azure subscription and d
 
 </details>
 
+## Add a configuration page to your messaging extension on GitHub
+
+[!include[v4-to-v3-SDK-pointer](~/includes/v4-to-v3-pointer-me.md)]
+
+## Code sample
+
+The Teams Search Auth Config for sample projects on GitHub, demonstrate how to create messaging extensions that include a configuration page and Bot Service authentication. The samples also demonstrate how to create message extensions that accept search requests and return the results after the user has signed in.
```

Comment:
Attached sample do not have authentication implemented, so we can remove **and Bot Service authentication** part or link mentioned here : https://github.com/microsoft/BotBuilder-Samples#teams-samples on Point 52.